server/tserver/src/main/java/org/apache/accumulo/tserver/AssignmentHandler.java

```diff
@@ -156,9 +156,7 @@ public void run() {
     Tablet tablet = null;
     boolean successful = false;
 
-    try {
-      server.acquireRecoveryMemory(extent);
-
+    try (var recoveryMemory = server.acquireRecoveryMemory(tabletMetadata)) {
```

Comment:
```suggestion
    try (var ignored = server.acquireRecoveryMemory(tabletMetadata)) {
```
The variable is unused so this prevents the IDE from complaining but not a big deal if you don't want to change it.

Reply:
This strategy doesn't work in all IDEs. Other strategies might work better to be more universal, like adding an assert statement to check that it's not null, or adding a trace log message or something for the variable.

Reply:
I don't think I made this change.  I meant to make the change and got side tracked and did not make it.  Asserting its not null seems like a nice thing to do.